### PR TITLE
Bulk Byte to Short Map Converter

### PR DESCRIPTION
**Pull request by Mark7625**
Added a way to Convert maps from Byte to Short in a Bulk format

### CODE REVIEW

File: Client/src/main/java/com/jagex/map/MapRegion.java [hunk 1]
```diff
@@ -478,6 +478,7 @@ public final void unpackObjects(SceneGraph scene, byte[] data, int localX, int l
 	}
 
 	public final void decodeOldMapData(Buffer buffer, int x, int y, int z, int regionX, int regionY, int orientation) {// XXX
+		System.out.println("Decode Oldd");
```
Comment (notmeta) [inline Client/src/main/java/com/jagex/map/MapRegion.java]: Remove

File: Editor/src/main/java/com/rspsi/controls/ConvertLandscapeTool.java [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+    	mapFolderText.setText("");
+    	okClicked = false;
+    	oldToNew.clear();
+    }
+
+    private Map<Integer, Integer> oldToNew = Maps.newConcurrentMap();
+
+	public void doRencode() {
+		File mapsFolder = new File(mapFolderText.getText());
+		System.out.println(mapsFolder);
```
Comment (notmeta) [inline Editor/src/main/java/com/rspsi/controls/ConvertLandscapeTool.java]: Remove(?)

File: Editor/src/main/java/com/rspsi/controls/ConvertLandscapeTool.java [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+					.filter(file -> !file.isDirectory())
+					.filter(FileUtils::isMapFile)
+					.forEach(file -> {
+						try {
+							byte[] data = Files.readAllBytes(file.toPath());
+							File outputFile = new File(outputDir, file.getName());
+							if (FileUtils.isDatOrGzFile(file)) {
+
+
+								System.out.println(file.getName());
```
Comment (notmeta) [inline Editor/src/main/java/com/rspsi/controls/ConvertLandscapeTool.java]: Remove(?)

File: Editor/src/main/java/com/rspsi/controls/ConvertLandscapeTool.java [hunk 4]
```diff
@@ -0,0 +1,378 @@
...
+			for(int x = 0;x<=width;x++) {
+				tileHeights[z][x][length] = tileHeights[z][x][length - 1];
+			}
+
+		}
+
+	}
+
+
+	public byte[] save_terrain_block(Chunk chunk) {
```
Comment (notmeta) [inline Editor/src/main/java/com/rspsi/controls/ConvertLandscapeTool.java]:
```suggestion
	public byte[] saveTerrainBlock(Chunk chunk) {
```

File: Editor/src/main/java/com/rspsi/controls/ConvertLandscapeTool.java [hunk 5]
```diff
@@ -0,0 +1,378 @@
...
+
+			}
+
+		}
+
+		byte[] data = Arrays.copyOf(buffer.getPayload(), buffer.getPosition());
+		return data;
+	}
+
+	private void save_terrain_tile(int level, int x, int y, Buffer buffer) {
```
Comment (notmeta) [inline Editor/src/main/java/com/rspsi/controls/ConvertLandscapeTool.java]:
```suggestion
	private void saveTerrainTile(int level, int x, int y, Buffer buffer) {
```

**Review by notmeta — CHANGES_REQUESTED** [review]

**Comment by Mark7625** [comment]
Sorted sorry about that I thought I removed all the prints

**Review by notmeta — APPROVED** [review]

**Pull request merged**